.github/workflows/post-merge-edge-node-container.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       contents: read
       packages: write
       deployments: write
-    uses: open-edge-platform/orch-ci/.github/workflows/post-merge.yml@main
+    uses: open-edge-platform/orch-ci/.github/workflows/post-merge.yml@0.1.8
```

Comment:
shall we use tag or shas ? In some cases I have seen using shas

Reply:
Used version/tag for pre-merge, works fine. 
Why SHAs are preferable?